Lesson27/Merve/create.js

```diff
@@ -0,0 +1,24 @@
+document.getElementById('newForm').addEventListener('submit', function (event) {
+    event.preventDefault();
+
+    const title = document.getElementById('title').value;
```

Comment:
In cases like this, don't forget that input values have data and that the data is in the expected format (e.g. at least check that the value is not less than 3 chars here and for the next input).